src/model/jugador/Jugador.java

```diff
@@ -0,0 +1,18 @@
+package model.jugador;
+
+import model.herramientas.Hacha;
+import model.herramientas.Herramienta;
+import model.materiales.Madera;
+import model.materiales.Material;
+
+public class Jugador {
+
+    protected Herramienta herramientaEquipada = new Hacha(new Madera());
+
```

Comment:
aca no deberias hacer el constructor? En vez de la linea 10, seria:

protected Herramienta herramienta;
public Jugador (){
      this.herramienta  = new Hacha(new Madera());

Reply:
Es cierto